apps/example/app/[locale]/layout.tsx

```diff
@@ -19,7 +19,7 @@ export default function Layout({
   }
 
   return (
-    <html lang={locale}>
+    <html lang={locale} suppressHydrationWarning={true}>
```

Comment:
Why ?

Reply:
Some browser extensions send data to NextJS, this flag allows us to not display related errors that are actually just warnings